### PR TITLE
Docs/initialization steps

### PR DESCRIPTION
**Pull request by knzeng-e**
This adds the initialization params to the .env_example.
It also updates the deploys-patron-kyc.js script to inject those values

### CODE REVIEW

File: scripts/deploy-patron-kyc.js [hunk 1]
```diff
@@ -38,13 +38,13 @@ const initMessage =
...
+  const end = process.env.END_DATE;
 
-  const ratio = ethers.utils.parseUnits("0.004", 18);
-  const hardCap = ethers.utils.parseUnits("10", "ether");
-  const contributionLimit = ethers.utils.parseUnits("0.5", "ether");
-  const patronRoles = [ethers.utils.namehash("email.roles.verification.apps.energyweb.iam.ewc")];
+  const ratio = ethers.utils.parseUnits(process.env.RATIO, 18);
+  const hardCap = ethers.utils.parseUnits(process.env.HARD_CAP, "ether");
+  const contributionLimit = ethers.utils.parseUnits(process.env.CONTRIBUTION_LIMIT, "ether");
+  const patronRoles = [ethers.utils.namehash(process.env.PATRON_ROLES)];
```
Comment (jrhender) [inline scripts/deploy-patron-kyc.js:47]: ROLE needs to be singular for this to work, I believe
```suggestion
  const patronRoles = [ethers.utils.namehash(process.env.PATRON_ROLE)];
```

Comment (knzeng-e) [reply scripts/deploy-patron-kyc.js:47]: oh yes correct. Thank you @jrhender

Comment (knzeng-e) [reply scripts/deploy-patron-kyc.js]: Impelemented on [f9492a5](https://github.com/energywebfoundation/staking-pool/pull/69/commits/f9492a53952e217d052c24499d3b94a584231097)

File: scripts/pool-deployment-env-files/.env_example_deploy [hunk 2]
```diff
@@ -3,4 +3,26 @@
...
+# 2 - Initialization parameters
+###########
+# Make sure to provide the right format
+# Ex:
+#    RATIO = 0.004
+#    HARD_CAP = 30
+#    END_DATE = 1734181200
+#    START_DATE = 1671022800 //UNIX FORMAT
+#    CONTRIBUTION_LIMIT = 2
+#    PATRON_ROLES = "email.roles.verification.apps.energyweb.auth.ewc"
```
Comment (jrhender) [inline scripts/pool-deployment-env-files/.env_example_deploy:18]: Needs to be fixed here too

Comment (knzeng-e) [reply scripts/pool-deployment-env-files/.env_example_deploy]: Updated [9ac1d97](https://github.com/energywebfoundation/staking-pool/pull/69/commits/9ac1d97125648e466d089b5ee0efb20417dad8ab). Thanks

**Review by jrhender — APPROVED** [review]

**Review by jrhender — APPROVED** [review]

**Pull request merged**